test/controllers/ScratchControllerSpec.scala

```diff
@@ -57,4 +65,29 @@ class ScratchControllerSpec extends WordSpec with Matchers with ScalaFutures wit
     }
 
   }
+
+  "Calling the get action" should {
+
+    "return an Ok response" in new Test {
+      private val result = target.get(uuid)(getRequest)
+      status(result) shouldBe OK
+    }
+
+    "return content as JSON" in new Test {
+      private val result = target.get(uuid)(getRequest)
+      contentType(result) shouldBe Some(ContentTypes.JSON)
+    }
+
+    "return a UUID assigned to an attribute labelled id" in new Test {
```

Comment:
Is this test description correct? It looks to me as if we are just checking the controller returns an expected Json object.